Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -245,13 +246,15 @@ class TouchableOpacity extends React.Component<Props, State> {
         nativeID={this.props.nativeID}
         testID={this.props.testID}
         onLayout={this.props.onLayout}
-        nextFocusDown={this.props.nextFocusDown}
-        nextFocusForward={this.props.nextFocusForward}
-        nextFocusLeft={this.props.nextFocusLeft}
-        nextFocusRight={this.props.nextFocusRight}
-        nextFocusUp={this.props.nextFocusUp}
+        nextFocusDown={tagForComponentOrHandle(this.props.nextFocusDown)}
+        nextFocusForward={tagForComponentOrHandle(this.props.nextFocusForward)}
+        nextFocusLeft={tagForComponentOrHandle(this.props.nextFocusLeft)}
+        nextFocusRight={tagForComponentOrHandle(this.props.nextFocusRight)}
+        nextFocusUp={tagForComponentOrHandle(this.props.nextFocusUp)}
         hasTVPreferredFocus={this.props.hasTVPreferredFocus === true}
-        isTVSelectable={this.props.isTVSelectable !== false && this.props.accessible !== false}
+        isTVSelectable={
+          this.props.isTVSelectable !== false && this.props.accessible !== false
```

Comment:
Unrelated, but this may be unexpected check, when 0 or "" is passed to these `boolean`s, maybe remove the strict equation check?

Reply:
@Krisztiaan thanks for the review!

- RN does already have `findNodeHandle`, however it is not allowed to be used in this context and will throw an error. I created my own method here and extracted it to its own file, and it will continue to change as the new architecture is rolled out.
- Those booleans have default values and should not be set to anything other than true/false, so I'd like to leave the strict equation checks :)